packages/core/src/mesh/Skin.ts

```diff
@@ -9,6 +9,10 @@ export class Skin extends EngineObject {
   public inverseBindMatrices: Matrix[];
   public joints: string[];
   public skeleton: string;
```

Comment:
`joints` and `skeleton` are not used now

Reply:
Keep readonly